paddle/fluid/operators/math/matrix_bit_code.h

```diff
@@ -159,7 +159,7 @@ class CustomCode : public Code {
 
     for (int i = 0; i < static_cast<int>(ptable_->dims()[1]); i++) {
       if (ptable_->data<R>()[index_ * static_cast<int>(ptable_->dims()[1]) +
-                             i] != -1) {
+                             i] >= 0) {
```

Comment:
this looks like more reasonable for users usage, all valid index should be greater than 0